content/dstu2.md

```diff
@@ -322,6 +322,28 @@ $ curl -i -H "Accept: application/json+fhir" "https://fhir-open.sandboxcerner.co
 In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status` 
  are passed in the query string.
 
+### Resource Identity
+ 
+Please note that, in order to determine the uniqueness of ids and identifiers in Millennium EHR, complete URL must be considered. A complete URL comprises of [service root url](http://hl7.org/fhir/dstu2/http.html#general) and parameters (if any).
+
+For example,
+
+To determine uniquesness of an id, consider:
+
+    https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person?_id=Patient/1316024
+
+instead of:
+
+    Patien/1316024
+
+To determine uniquesness of an identifier, consider:
```

Comment:
Take a look over Jenni's comment again about not guaranteeing uniqueness of identifiers.